p2panda-core/src/operation.rs

```diff
@@ -128,6 +128,32 @@ where
     pub fn hash(&self) -> Hash {
         Hash::new(self.to_bytes())
     }
+
+    /// Number of fields included in the header.
+    ///
+    /// Fields instantiated with `None` values are excluded from the count.
+    pub fn len(&self) -> usize {
```

Comment:
Assuming we go with this change: is `len()` the best name?

Could also be `count_elements()` or similar.